recipe-server/client/control_new/components/common/LoadingOverlay.js

```diff
@@ -0,0 +1,29 @@
+import { Spin } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { areAnyRequestsInProgress } from 'control_new/state/requests/selectors';
+
+
+@connect(
+  state => ({
+    loading: areAnyRequestsInProgress(state) || false,
```

Comment:
I don't think you need the `|| false`, since `loading` is tested for truthiness later and doesn't necessarily need to be a bool.

Beyond that, I'd sort of expect `areAnyRequests` to return false so we don't need to `|| false`, but I haven't looked at that code yet.